test/tests/ports/features.spec.ts

```diff
@@ -1,13 +1,11 @@
-import fetch, { Response } from 'node-fetch'
+import { Response } from 'node-fetch'
 import { IFetchComponent } from '@well-known-components/http-server'
+import { createFeaturesComponent } from '@well-known-components/features-component'
 import {
   IConfigComponent,
   ILoggerComponent,
 } from '@well-known-components/interfaces'
-import {
-  IFeaturesComponent,
-  createFeaturesComponent,
-} from '../../../src/ports/features'
+import { IFeaturesComponent } from '@well-known-components/features-component/dist/types'
```

Comment:
This file should be deleted, right? We're importing the features component from the well-known component repo.

Reply:
you're right, ty for the review! removing it